src/Hacks/aimbot.cpp

```diff
@@ -61,6 +62,8 @@ QAngle RCSLastPunch;
 
 int Aimbot::targetAimbot = -1;
 
+static xdo_t *xdo = xdo_new(NULL);
```

Comment:
Maybe it can fix the mouseup issue. 
And what about creating a bool to release the key next tick?